manim/mobject/svg/svg_mobject.py

```diff
@@ -218,12 +221,70 @@ def attribute_to_color(attr: str) -> Color:
         return Color(attr)
 
     @staticmethod
-    def css_style_to_style(attr: str):
+    def apply_css_style(vmobject: VMobject, attr: str) -> VMobject:
+        """Converts inline CSS styles and applies it to the provided :class:`~.VMobject`.
+        If the property name is recognized (e.g. 'fill: red'), then it is mapped to its
+        corresponding :class:`~.VMobject` style property (e.g. 'fill_color').
+        Unrecognized CSS properties (e.g. 'stroke-miterlimit: 1') are ignored.
+
+        Parameters
+        ----------
+        vmobject : :class:`~.VMobject`
+            The VMobject to apply the style to
+        attr : :class:`string`
+            The value of the SVG 'style' attribute to convert
+
+        Returns
+        ----------
```

Comment:
This is valid, but.... just a suggestion
```suggestion
        -------
```